django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -155,7 +155,6 @@
 
 #changelist-filter ul:last-child {
     border-bottom: none;
-    padding-bottom: none;
```

Comment:
I think this should be changed to `padding-bottom: 0;` :thinking:

Reply:
I'm fine either way. I think the visuals look good with and without the padding. The padding has existed for 5 years since 35901e64b043733acd1687734274553cf994511b, so I figure it can't be too noticeable.

Reply:
@elky Can you take a look?

Reply:
Good find, @jdufresne

In terms of visual look I think it's better to remove this line and add `margin-bottom: 0` to make gaps look consistent with "not-last" sections.

<s>`padding-bottom: none;`</s>
`margin-bottom: 0;`

<img width="254" alt="Screenshot 2020-05-04 at 23 15 08" src="https://user-images.githubusercontent.com/209663/81009328-1bef6d00-8e5d-11ea-8b91-44232b66076d.png">

vs

`padding-bottom: 0`


<img width="288" alt="Screenshot 2020-05-04 at 23 12 02" src="https://user-images.githubusercontent.com/209663/81009102-ad121400-8e5c-11ea-9dfc-8fc82d0ff590.png">

Reply:
Isn't this the same visual effect as dropping the line entirely? (The original patch.) The existing padding rule already adds the spacing. I think I should go back to dropping it. Here is the result of removing the line:

![remove-padding](https://user-images.githubusercontent.com/347634/81020943-dad86800-8e1e-11ea-9ea9-89a674b8085e.png)